common/src/main/java/com/microsoft/identity/common/internal/controllers/ExceptionAdapter.java

```diff
@@ -158,39 +154,54 @@ private static boolean shouldBeConvertedToUiRequiredException(final String oAuth
 
     }
 
+    private static boolean isIntunePolicyRequiredError(final String oAuthError, final String oAuthSubError) {
+
+        return  !TextUtils.isEmpty(oAuthError) &&
+                !TextUtils.isEmpty(oAuthSubError) &&
+                oAuthError.equalsIgnoreCase(AuthenticationConstants.OAuth2ErrorCode.UNAUTHORIZED_CLIENT) &&
+                oAuthSubError.equalsIgnoreCase(AuthenticationConstants.OAuth2SubErrorCode.PROTECTION_POLICY_REQUIRED);
```

Comment:
The sub error PROTECTION_POLICY_REQUIRED is for True MAM CA. I am wondering if we could have a better naming here.

Reply:
I was following the changes by the Intune team member here in adal and used the same name to avoid confusion : https://github.com/AzureAD/azure-activedirectory-library-for-android/blob/dev/adal/src/main/java/com/microsoft/aad/adal/IntuneAppProtectionPolicyRequiredException.java

Any recommendations ? Also any reason you thought this is not apt?